packages/core/logger/src/Logger.js

```diff
@@ -67,7 +67,7 @@ let consolePatched = false;
 // TODO: Implement the rest of the console api as needed.
 // TODO: Does this need to be disposable/reversible?
 export function patchConsole() {
-  if (consolePatched) {
+  if (consolePatched || process.env.NODE_ENV === 'test') {
```

Comment:
How would I achieve this without breaking the tests?

Setting the option `patchConsole: false` does not seem to really do anything

Reply:
I'm willing to work on this if this would be something anyone wants.

Reply:
Yeah we should probably make the `patchConsole` option also apply in workers... @wbinnssmith wdyt?